### PR TITLE
Add support for Django 4.2

### PR DESCRIPTION
**Pull request by browniebroke**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -45,6 +45,7 @@ def rel(*parts):
     'Framework :: Django :: 3.2',
     'Framework :: Django :: 4.0',
     'Framework :: Django :: 4.1',
+    'Framework :: Django :: 4.2',
```
Comment (browniebroke) [inline setup.py:48]: That's blocked by https://github.com/pypa/trove-classifiers/pull/128

Comment (fjsj) [reply setup.py:48]: Will this break the build?

Comment (browniebroke) [reply setup.py:48]: I _think_ it'll prevent any releases

Comment (fjsj) [reply setup.py:48]: OK, I'll wait for the fix in upstream than. Thanks for informing.

Comment (browniebroke) [reply setup.py:48]: No longer blocked now 🎉

**Review by foarsitter — APPROVED** [review]

**Comment by browniebroke** [comment]
Do you think that this could be merged? Django 4.2 has been released now

File: tests/requirements/django42.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+django>=4.2a1,<5.0
```
Comment (andruten) [inline tests/requirements/django42.txt:1]: I would change this to >=4.2,<5 for coherence with other versions. What do you think?

Comment (browniebroke) [inline tests/requirements/django42.txt:1]:
```suggestion
django>=4.2,<5.0
```

Comment (browniebroke) [reply tests/requirements/django42.txt]: Good point! Changed

**Review by foarsitter — APPROVED** [review]

**Pull request merged**